src/app/content/App/Contribute/ContributeScreen/SubmitContributionForm/SubmitContributionForm.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Field, InjectedFormProps } from 'redux-form';
+import withReduxForm from './withReduxForm';
```

Comment:
It does not seem to work … relative imports should be after absolute ones (https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md)

Reply:
I think we just need to tweak the rule a bit.

So I guess your fine with this setting? No changes?
```
// 1. node "builtin" modules
import fs from 'fs';
import path from 'path';
// 2. "external" modules
import _ from 'lodash';
import chalk from 'chalk';
// 3. "internal" modules
// (if you have configured your path or webpack to handle your internal paths differently)
import foo from 'src/foo';
// 4. modules from a "parent" directory
import foo from '../foo';
import qux from '../../foo/qux';
// 5. "sibling" modules from the same or a sibling's directory
import bar from './bar';
import baz from './bar/baz';
// 6. "index" of the current directory
import main from './';
// 7. "object"-imports (only available in TypeScript)
import log = console.log;
```
@gregoirelacoste comments?

Reply:
Yes (default one, right ?) it’s what I was doing anyway …

I’d like to have the `4.` be more specific with outer folders first, but I’m tinkering

Now, we just have to make it work like expected :-)

Reply:
@lutangar Nice! I usually import react and its modules always first, we can immediately see if we are in a component, it might be good to put it first when possible?

Reply:
> import react and its modules always first,

Not sure I can enforce this with this rule, but if put first the react import will stay first.

@JalilArfaoui the config is now working as expected

Reply:
ping @JalilArfaoui